src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao = new DriverDaoImpl();
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        if (driverDao.get(id).isEmpty()) {
+            throw new DataProcessingException("Couldn't get driver by id " + id);
+        }
+        return driverDao.get(id).get();
```

Comment:
```suggestion
        return driverDao.get(id).orElseThrow();
```
better do it this way, you can also pass some exception with informative message into `orElseThrow()`